fonts.css

```diff
@@ -0,0 +1,15 @@
+@font-face {
+	font-family: 'Gotham Pro';
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/ProjectMod2_EKh/pullRequest?prid=6247910)